src/main/java/com/faforever/client/mod/Mod.java

```diff
@@ -19,7 +18,7 @@
   private final StringProperty author;
   private final ObjectProperty<OffsetDateTime> createTime;
   private final ObjectProperty<OffsetDateTime> updateTime;
-  private final ObjectProperty<Player> uploader;
+  private final StringProperty uploader;
```

Comment:
Why move to String?

Reply:
I moved to string for consistency with author and also since only the username is needed